src/less/charts.less

```diff
@@ -98,13 +98,19 @@
 //Responsive styles for mobiles
 
 @media (max-width: @global-screen-xsmall-max) {
-  .chart-doughnut__info {
-    div {
-      margin-top: @global-whitespace-small !important;
-    }
+  .chart {
+    .chart__header {
+      display: block !important;
```

Comment:
Let's make sure !important is always our last resort since they mess up specificity for the codebase moving forward.

In this case the reason it's necessary is that we're using a .u-flex class on the chart__header div in the react component. If you remove that utility class and instead add display:flex in the .chart__header class, you will be able to then change that to a display:block on the media query without needing to use !important.